.editorconfig

```diff
@@ -0,0 +1,12 @@
+[*]
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Should we also add `max_line_length = 120`? This is already covered by clang-format, but I think this will make in-IDE formatters closer to our style while editting.

Reply:
Good idea, I added it